backend/lib/endpoints/posts.js

```diff
@@ -52,18 +52,30 @@ async function routes(app) {
           limit,
           objective,
           skip,
+          postMode,
           includeMeta,
         },
       } = req;
+      console.log("postProps modeURL", objective, postMode);
       const queryFilters = filter ? JSON.parse(decodeURIComponent(filter)) : {};
       // Base filters - expiration and visibility
       /* eslint-disable sort-keys */
-      const filters = [
+      let filters = [
         {
           $or: [{ expireAt: null }, { expireAt: { $gt: new Date() } }],
           status: { $ne: "removed" },
         },
       ];
+      // "IA" stands for InActive or Archived
+      //In future we will also have "D" for Drafts
+      if(postMode === 'IA'){
```

Comment:
since it's just started to add these states, maybe just use the whole word, e.g. 'INACTIVE', later can have 'ARCHIVED', etc.